features.json

```diff
@@ -78,13 +78,14 @@
 			"logo": "/images/firefox.svg",
 			"version": "77",
 			"features": {
-				"bigInt": "Nightly",
+				"bigInt": "Beta 78",
 				"bulkMemory": "javascript.options.shared_memory",
-				"multiValue": "Nightly",
+				"multiValue": "Beta 78",
 				"mutableGlobals": true,
 				"referenceTypes": "Nightly",
 				"saturatedFloatToInt": true,
 				"signExtensions": true,
+				"simd": "Nightly, x86 only",
```

Comment:
Just in case - is this w/o flags?

Reply:
there is `javascript.options.wasm_simd` flag (in the Nightly version only) which is enabled by default

Reply:
Sigh, okay. Another clarification - where is information about "x86" coming from? Does it include x86-64? It's not very obvious IMO if so.

Reply:
Judging by dependat bugs of [an umbrella SIMD bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1625130), x86/x64 support is ready, arm32 and arm64 is work in progress. AFAIK, arm support landed in Cranelift couple weeks ago.

Do you want me to replace x86 by x86/x64? I meant the [x86 family](https://en.wikipedia.org/wiki/X86) which includes 16-bit, 32-bit and 64-bit versions, but I agree that this is not obvious. I'd be happy to make it more clean if necessary.

Reply:
Well Firefox likely doesn't run on 16-bit platforms, and 64-bit is traditionally encoded as x86-64 to explicitly distinguish from regular x86. You can see even in the link you provided `x86` only includes 16-bit and 32-bit versions, with `x86-64` being a separate group of rows.

Let's do `x86 and x86-64 only`.

Reply:
> AFAIK, arm support landed in Cranelift couple weeks ago.

This is not correct.  Some arm64 work is ongoing in Cranelift, which has a largely functioning arm64 backend.  No arm(32) work is happening in Cranelift to my knowledge.

Even though the matter has been resolved to my satisfaction, for the sake of record: IMO we should always record x86 and x64 aka x86-64 separately, as they are separate if closely related back-ends.